alternator/executor.cc

```diff
@@ -562,10 +563,11 @@ future<> verify_permission(
     // path (when the operation is allowed). Using a continuation instead of
     // co_await makes it faster (no allocation) in the happy path where
     // permissions are cached and check_has_permissions() doesn't yield.
-    return client_state.check_has_permission(auth::command_desc(
-            permission_to_check,
-            auth::make_data_resource(schema->ks_name(), schema->cf_name()))).then(
-        [permission_to_check, &schema, &client_state] (bool allowed) {
+    return do_with(auth::make_data_resource(schema->ks_name(), schema->cf_name()),
```

Comment:
would be better to fix by corutiniazation than do_with

Reply:
Done